enhancements/ingress/custom-route-configuration.md

```diff
@@ -193,6 +194,25 @@ pairs.
 Library-go is also an ideal spot to provide functionality to inject route hosts.  Something like our StaticResource
 controller which automatically handles the API status and the required route changes.
 
+#### Trusting the oauth-server
+
+Some OpenShift components need to communicate with the OpenShift oauth-server in order to
+retrieve access tokens on behalf of the user that wants to use them. Typical examples include
+the web console and anything that uses the oauth-proxy, like the monitoring tools. A custom oauth-server
+serving certificate might break trust from the other components in the cluster.
+
+The authentication operator needs to publish the oauth-server's  trust root in a well-known location so
+that components that need to trust the oauth-server can use it. The authentication operator will therefore
+always create a configMap `openshift-config-managed/oauth-server-cert` which is going to include:
+
+1. the closest-to-leaf CA certificate to the oauth-server serving certificate if that is possible
+2. the oauth-server's serving certificate if no CA certificate that signed it is known
```

Comment:
@slaskawi question: @stlaz and me brainstormed if this is actually too complex. Instead, we could simplify this and just return the configured serving cert as-is. Do you see pros/cons?

Reply:
I think the main reason to use CAs as far away from root as possible is helpful for blacklisting. This way, when you invalidate CAs close to the leaf, there's no reason to regenerate certs between the leaf and the root. 

But I guess there's no CA blacklisting mechanism implemented in OpenShift whatsoever. If my assumption is correct - I guess it's safe to use the serving cert and simplify this part.

Reply:
modified so that only the serving-cert is present